packages/homepage/content/articles/new-dashboard/index.md

```diff
@@ -0,0 +1,93 @@
+---
+banner: ./images/banner.png
+slug: new-dashboard
+authors: ["Ives van Hoorne"]
+photo: https://avatars0.githubusercontent.com/u/587016?s=460&v=4
+title: A New-look Dashboard
```

Comment:
This implies that we only did a redesign of the dashboard, but in reality I think we've done more than that. Maybe it would make sense to set as title: "Introducing Workspaces with a new Dashboard" or just "Introducing Workspaces"? That way we revolve the release around workspaces, which I think is the most important part of the redesign of the dashboard, it's a new dashboard that's built for collaboration.